spec/requests/api_spec.rb

```diff
@@ -0,0 +1,285 @@
+require 'rails_helper'
+
+describe Api, :pagination_helper do
+  before(:all) do
+    @user = create_really_basic_user
+    @api_token = @user.create_api_token!
+    @auth_header = { 'Littlesis-Api-Token': @api_token.token }
+  end
+
+  after(:all) do
+    @user.sf_guard_user.delete
+    @user.delete
+    @api_token.delete
+  end
+
+  let(:meta) do
+    {
+      'copyright' => Api::META[:copyright],
+      'license' => Api::META[:license],
+      'apiVersion' => Api::META[:apiVersion]
+    }
+  end
+
+  describe 'entities' do
+    let(:lawyer) do
+      create(:entity_person).tap { |e| e.add_extension('Lawyer') }
+    end
+
+    let(:expected) do
```

Comment:
this is interesting. you define a ruby hash here, but your assertion on line 56 claims that the json coming back from the api will equal this hash. is rspec performing hash -> json conversion implicitly for free? or something i'm not seeing you wrote to perform that conversion / equality comparison yourself?

Reply:
we are doing the JSON parsing ourselves in rspec via a helper function ```json```: https://github.com/public-accountability/littlesis-rails/blob/api/spec/support/request_macros.rb